sdk/schemaregistry/azure-schemaregistry/azure/schemaregistry/_generated/aio/operations/_operations.py

```diff
@@ -414,9 +212,9 @@ async def get_schema_version(
             deserialized = response.iter_bytes()
 
         if cls:
-            return cls(pipeline_response, cast(AsyncIterator[bytes], deserialized), response_headers)
+            return cls(pipeline_response, cast(AsyncIterator[bytes], deserialized), response_headers)  # type: ignore
 
-        return cast(AsyncIterator[bytes], deserialized)
+        return cast(AsyncIterator[bytes], deserialized)  # type: ignore
 
     async def query_id_by_content(  # pylint: disable=inconsistent-return-statements
```

Comment:
when a cls is passed in here, it doesn't return None, so is the typing wrong? Or is there a way to handle typing on the returned object.?

Reply:
type: ignore in return